01_fib/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote("1\n2\n3\n5\n8\n13\n21\n")
```

Comment:
This should print 1\n1\n2\n3\n5\n8\n13. Please check the implementation.

Reply:
Done.